src/Microsoft.DotNet.Interactive/KernelInvocationContext.cs

```diff
@@ -62,14 +62,18 @@ private KernelInvocationContext(KernelCommand command)
                     c.Error.Subscribe(s => this.DisplayStandardError(s, command))
                 };
             }));
-        
+
             _disposables.Add(operation);
         }
 
         public KernelCommand Command { get; }
 
         public bool IsComplete { get; private set; }
 
+        private bool _isFailed;
```

Comment:
Nitpick: Move field declarations to the top of the class.

Reply:
Done